lib/core-data/get-variations.js

```diff
@@ -0,0 +1,104 @@
+import _ from 'lodash';
+import logger from '../utils/log';
+
+const log = logger(__filename);
+
+/*
+ * adds a 'componentVariation' field to the components schema
+ * @param {schema} object
+ * @param {componentVariations} array
```

Comment:
Not the correct way to document functions

Reply:
had a brain fart, fixed